indexer/jobs/export_coin_balances_job.py

```diff
@@ -30,7 +30,7 @@ class AddressRecord:
 
 # Exports coin balances
 class ExportCoinBalancesJob(BaseExportJob):
-    dependency_types = [Block, ContractInternalTransaction]
+    dependency_types = [Block, Transaction, ContractInternalTransaction]
```

Comment:
Do you need this?

Reply:
yes.
Based on the logic in line 50 of this file, I fixed the missing Transaction bug in dependency_types.

Reply:
Considering the original code's intention of omitting dependencies and the improvements that will be brought in another PR, I will roll back the changes to this line of code and modify the code in _collect.